doc/fileexplorer.rst

```diff
@@ -1,23 +1,84 @@
 #####
-Files
+Files 
 #####
 
-The **Files** pane is a built-in filesystem and directory browser allowing you to view and filter files and their properties, open them with the :doc:`editor` or an external tool, and perform common management tasks (varies by platform).
+The **Files** pane is a filesystem and directory browser built right into Spyder.
+You can view and filter files according to their type and extension, open them with the :doc:`editor` or an external tool, and perform many common operations.
 
 .. image:: images/files/files-standard.png
-   :alt: Spyder File Explorer panel, showing a tree view of files and metadata
+   :alt: Spyder Files pane, showing a tree view of files and metadata
 
 
 
-==========================
-Performing file operations
-==========================
+===============
+File operations
+===============
 
-You can use context menus to access a number of functions on a per-file and per-directory basis, including running scripts, creating, renaming, moving, editing or deleting files, performing basic operations with ``git``, and opening an external file explorer.
+To browse the files on your system, you can use the arrows at the top of the pane.
+You can expand/collapse the folders in the pane to display the files and subdirectories hierarchically. 
+Double-clicking a folder will open it, showing the files inside and making it your working directory.
 
-.. image:: images/files/files-contextmenu-new.png
-   :align: center
-   :alt: File Explorer context menu, with common file manipulation tasks
+.. image:: images/files/files-browse.gif
+   :alt: Spyder Files pane showing browsing directories
+
+
+To open a file in the :guilabel:`Editor` from the :guilabel:`Files` pane, double-click its name.
+If you right-click over it, you will see a context menu that allows you to access a number of functions, including running scripts; creating, renaming, moving, deleting files; and opening them in your computer's file manager.
+
+.. image:: images/files/files-context-menu.png
```

Comment:
I would suggest cropping the screenshot you already have much more tightly on the context menu here so it is easier to read and focus on and takes up less unnecessary space, since the purpose of the screenshot is to show the  context menu, not the rest of the full Spyder UI.

Example of roughly what I'm thinking:

![image](https://user-images.githubusercontent.com/17051931/99687567-6d9ec400-2a4a-11eb-8cb4-fecf23778582.png)

Reply:
I agree with @CAM-Gerlach.

Reply:
Redoing this but showing the menu inside the files pane